### ISSUE TITLE
Findability] User can't determine where a page is within the VA.gov website.

### ISSUE THREAD

**Issue by erinrwhite** [erinrwhite/stagingreview-mods#12]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Testing team
**Product:** Testing product
**Feature:** Testing feature

## Findings details
**VA.gov Experience Standard - issue:** User can't determine where a page is within the VA.gov website.
**VA.gov Experience Standard - category:** Findability
**Launch-blocking:** Yes
**Collab Cycle Reviewer:** @erinrwhite (IA)

### Description

Once in the form flow, the breadcrumbs do not accurately convey where the user is within VA.gov. The breadcrumbs do not include a link back to the root of the app, resulting in a loss of wayfinding context.

## Recommended action
Update the breadcrumbs to reflect the name of the tool/app as the ""current"" page, staying consistent across all pages in the flow: `Home > Learn how the PACT Act may affect you' This is similar to how we do breadcrumbs on other forms, and it is necessary for users to understand what tool they are using with this simplified one-thing-per-page model. This pattern is effective on the income limits wizard.

## References

- **WCAG Success Criteria:** [WCAG 3.2.3 Consistent Navigation - Level AA](https://www.w3.org/TR/WCAG21/#consistent-navigation)
- **Modality:** N/A
- **Design System Component:** [Breadcrumbs ](https://design.va.gov/components/breadcrumbs)
- **Design System Pattern:** N/A
- **Content Style Guide:** [Writing for SEO](https://design.va.gov/content-style-guide/seo)
- **Context:** Navigation
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
